src/RpcServer/RpcServer.Wallet.cs

```diff
@@ -198,7 +198,7 @@ protected virtual JToken SendFrom(JArray _params)
             AssetDescriptor descriptor = new(snapshot, system.Settings, assetId);
             BigDecimal amount = new(BigInteger.Parse(_params[3].AsString()), descriptor.Decimals);
             if (amount.Sign <= 0)
-                throw new RpcException(-32602, "Invalid params");
+                throw new RpcException(RpcErrorFactory.InvalidParams("Amount can't be negative."));
```

Comment:
L156 in this file: `WalletNotFound` error instead of `FileNotFoundException`.

L158 in this file: `WalletNotSupported` error instead of NotSupportedException.